### PR TITLE
Upgrade foundation 6.3

### PR DESCRIPTION
**Pull request by sineade**
:art:

upgraded ama_styles to newest version of foundation 6.3.1.0
Fixed mixin grid because foundation made some changes that deprecated the code we were using.
had to change $base-font-size to a pixel value because with the rem value it kept throwing an error.
Added some styling to hide the automatic styling the drop down forms by the browser so we could display our custom styling.
After updating the space below the header changed, so I had to adjust the value of top-nav-offset.

### CODE REVIEW

**Comment by sineade** [comment]
:hankey: this is ongoing while I continue to update the other apps and find bugs from the upgrade.

**Comment by sineade** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

File: app/assets/stylesheets/_settings.scss [hunk 1]
```diff
@@ -521,6 +521,7 @@ $tab-content-background: $white;
 $tab-content-border: none;
 $tab-content-color: foreground($tab-background, $primary-color);
 $tab-content-padding: 0;
+$tab-color: #ffffff;
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/_settings.scss]: Should this be `$white`?

Comment (sineade) [inline app/assets/stylesheets/_settings.scss]: yes!

**Comment by sineade** [comment]
:wave:

File: app/assets/stylesheets/_settings.scss [hunk 2]
```diff
@@ -59,7 +59,7 @@ $light-gray: #e6e6e6;
 $medium-gray: #cacaca;
 $dark-gray: #8a8a8a;
 $black: #0a0a0a;
-$white: #ffffff;
+$white: $white;
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/_settings.scss]: This probably won't work...

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

**Pull request merged**